pages/styles/style.css

```diff
@@ -0,0 +1,288 @@
+html {
+    font-size: 16px;
+    background-color: rgb(228, 222, 222);
+}
+
+#home-header {
+    font-size: 2.5rem;
+    font-family:Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    color:rgb(106, 135, 135);
+    text-align: center;
+    padding:8%;
+    padding-bottom: 4%;
+    background-image: url(https://images.rawpixel.com/image_1000/czNmcy1wcml2YXRlL3Jhd3BpeGVsX2ltYWdlcy93ZWJzaXRlX2NvbnRlbnQvbHIvcm0zMDUtbnVubnktMDZhLWwuanBn.jpg);
+    background-origin: content-box;
+}
+
+
+header ul {
+    list-style-type: none;
+ }
+
+body{
+	font-size:1.5rem;
+    line-height:1.75; 
+    font-family:system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+}
+
+#Wanjun_photo {
+    width: 200px;
+    height: 210px;
+    border-radius: 70%;
+}
+
+#About-me-logo {
+    width: 500px;
+    height: 200px;
+}
+
+#portfolio-button {
+    width: 200px;
+    height: 100px;
+}
+
+.me-photo {
+    max-width: 260px;
+    max-height: 300px;
+}
+
+.logo {
+    width:40px;
+    height:40px;
+} 
+
+#divider {
+    width: 1500px;
+}
+
+.project-photo {
+    width:20%;
+    padding-left: 40%;
+}
+
+a {
+    color:rgb(134, 42, 42);
+}
+
+a:hover {
+    color:rgb(221, 151, 151);
+}
+
+a {
```

Comment:
There is already an `a` CSS style rule on line 63. These rules can be added there to make things DRY.